server/src/organizations/entities/organization.entity.ts

```diff
@@ -40,7 +41,10 @@ export class Organization {
 
   @OneToMany(
     () => Transaction,
-    (transaction) => transaction.donater_organization || transaction.recipient,
+    (transaction) => transaction.asset.posterOrganization || transaction.recipient,
```

Comment:
So, is this (`transaction.asset.posterOrganization || transaction.recipient`) saying that it could have a relationship to either the asset or the transactions tables at anytime?

Reply:
> So, is this (`transaction.asset.posterOrganization || transaction.recipient`) saying that it could have a relationship to either the asset or the transactions tables at anytime?
it's saying that an org can have transactions via an org's employees posting or via claiming an asset. so i think that's a yes to your question?

> This, I believe creates a relationship from `Org` to `Transactions` via transactions.recepientId
correct, i think that's what we want